services/help-service.ts

```diff
@@ -93,7 +126,7 @@ export class HelpService implements IHelpService {
 	}
 
 	// This method should return Promise in order to generate all html pages simultaneously.
-	private async createHtmlPage(basicHtmlPage: string, pathToMdFile: string): Promise<void> {
+	private async createHtmlPage(basicHtmlPage: string, pathToMdFile: string, pathToMdPages: string, pathToHtmlPages: string, extensionName?: string): Promise<void> {
```

Comment:
Not a merge-stopper as this is a `private` method, but I'd rather we didn't have as many arguments when we could do with a single object